test/db.connection.test.ts

```diff
@@ -0,0 +1,26 @@
+import assert from "assert";
+import {connect, getState, closeConn} from "../config/db.server.config";
+
+/**
+ * Test the database connection
+ */
+describe("Database connection test", function() {
+    it("should connect to and disconnect from the database with correct statuses", async function() {
+            const testDatabaseName = process.env.DATABASE_TEST_NAME;
```

Comment:
This variable looks like it is never used

Reply:
yeah as per our meeting we will be ignoring tests until they can be reviewed and fixed

Reply:
gotcha

Reply:
they're very broken and dependant on a prod database. I have started some initial work on fixing them but we want to get this in first

Reply:
yikes. thats good and makes sense though. thanks for the heads up